src/editor/FluxQueryField.tsx

```diff
@@ -42,6 +29,13 @@ function expandQuery(bucket, measurement, field) {
   );
 }
 
+class FluxSuggestion extends Suggestion {
```

Comment:
Why is this needed?

Reply:
For various reasons:
- I was trying to follow this "override pattern" that I noticed with other plugins. 
- I noticed a difference between a mouse click, and hitting 'tab' or 'enter' on a suggestion. You got fundamentally different objects. This was an attempt to wrap them together